### PR TITLE
fix xonsh links

### LINKED ISSUE #13: bad link on front page of docs
The "Documentation" link points to `xon.sh` instead of Rickshaw docs

### PR DESCRIPTION
**Pull request by scopatz**
Should fix #13 CC @gonuke @FlanFlanagan

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -211,7 +211,7 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'xonsh.tex', u'xonsh documentation',
+  ('index', 'xonsh.tex', u'rickshaw documentation',
```
Comment (gonuke) [inline docs/conf.py]: Should the 2nd entry still be `xonsh.tex`?

**Review by gonuke — CHANGES_REQUESTED** [review]
Still looks like there is one lingering reference to xonsh

File: docs/conf.py (hunk 1 above)
Comment (scopatz) [reply docs/conf.py]: Whoops! fixed!

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
LGTM - thanks @scopatz

**Comment by gonuke** [comment]
Does this automatically get deployed? Or do I need to (figure out how to) do that?

**Comment by scopatz** [comment]
It seems that doctr is set up on this repo, so it should have already been deployed.

**Comment by gonuke** [comment]
cool - this is my first look at `doctr`...  I can't help but feel like new docs will be pushed by `doctr` for every PR... it that correct?  intended?

**Comment by scopatz** [comment]
Yes, I think that is what is happening. I think the idea is that the docs are built for every PR (as a test) and then when the PR is merged, the docs are built again and actually deployed (pushed to the github pages repo).

**Comment by gonuke** [comment]
But it appears that the docs are deployed each time a PR is tested... or is doctr wise enough to avoid that?

**Comment by scopatz** [comment]
I think it depends on how it is setup, and it may not be set up correctly here. @asmeurer @gforsyth and @CJ-Wright might have a better answer to that question.

**Comment by CJ-Wright** [comment]
I think doctr knows if it is building on the master branch and doesn't push unless it is on the master branch, unless you have some special settings.
You could just test this by putting up a trial PR and see if it shows up on the webpage.

**Comment by asmeurer** [comment]
doctr only pushes from master unless `--no-require-master` is used (it isn't being used in this repo), and even then, it is impossible to push from a pull request from a fork.

**Comment by asmeurer** [comment]
And yes, in a PR, doctr runs all the steps of the deploy sans the push in every build to ensure that they would work. But the push is not done (see https://travis-ci.org/cyclus/rickshaw/builds/347951348#L1373). It seems we need to color the doctr output better.

**Pull request merged**